### DISCUSSION TITLE
Format columns in a table

### DISCUSSION THREAD

**Discussion by bookoutc34** [dfinke/ImportExcel#1377] [Q&A] [answered]
Hi,

How do i highlight a range of cells for just my table? I am using this command and it is highlighting the entire worksheet.

Add-ConditionalFormatting -Worksheet $ws -Range F:F -Value "Computers"  -RuleType LessThan -ConditionValue "25" -BackgroundColor Red

**Comment by dfinke** [comment, marked as answer]
Please post a short repro with data

**Reply by bookoutc34** [reply]
This is an example of what i am trying to do.
$myFile = "c:\temp\Test File"+(Get-Date -Format "yyyy-MM-dd HH-mm-ss")+".xlsx"
$data = ConvertFrom-Csv @"
Region,State,Units,Price
West,Texas,927,923.71
North,Tennessee,466,770.67
East,Florida,520,458.68
East,Maine,828,661.24
West,Virginia,465,053.58
North,Missouri,436,235.67
South,Kansas,214,992.47
North,North Dakota,789,640.72
South,Delaware,712,508.55
"@
$xl = $data |  Export-Excel -Path $myFile -AutoSize -TableName Breakdown -WorksheetName Sales -FreezeTopRow -BoldTopRow
$ws = $xl.Sales
Add-ConditionalFormatting -Worksheet $ws -Range C:C   -RuleType LessThan -ConditionValue "250" -BackgroundColor Red
Close-ExcelPackage -Show $xl

**Reply by dfinke** [reply]
## Try

```powershell
$data = ConvertFrom-Csv @"
Region,State,Units,Price
West,Texas,927,923.71
North,Tennessee,466,770.67
East,Florida,520,458.68
East,Maine,828,661.24
West,Virginia,465,053.58
North,Missouri,436,235.67
South,Kansas,214,992.47
North,North Dakota,789,640.72
South,Delaware,712,508.55
"@

$myFile = "$PSScriptRoot\spike.xlsx"
Remove-Item $myFile -ErrorAction SilentlyContinue

$xl = $data | Export-Excel -Path $myFile -AutoSize -TableName Breakdown -WorksheetName Sales -FreezeTopRow -BoldTopRow -PassThru

$ws = $xl.Sales

$range = "C1:C$($ws.Dimension.Rows)"

Add-ConditionalFormatting -Worksheet $ws -Range $range -RuleType LessThan -ConditionValue "250" -BackgroundColor Red

Close-ExcelPackage -Show $xl
```

**Reply by bookoutc34** [reply]
Worked perfectly. Thanks
